src/Video.tsx

```diff
@@ -154,7 +154,12 @@ const Video = forwardRef<VideoRef, ReactVideoProps>(
 
         const isLocalAssetFile =
           typeof _source === 'number' ||
-          ('uri' in _source && typeof _source.uri === 'number');
+          ('uri' in _source && typeof _source.uri === 'number') ||
+          ('uri' in _source &&
+            typeof _source.uri === 'string' &&
+            (_source.uri.startsWith('file://') ||
+              _source.uri.startsWith('content://') ||
+              _source.uri.startsWith('./')));
```

Comment:
I think . is enough as the file can be in '../forlder'

```suggestion
              _source.uri.startsWith('.')));
```